# Publish all joint states -----------------------------------
joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50  

# Wheels controllers -----------------------------------
wheel1_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: wheel1_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}

wheel2_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: wheel2_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}

wheel3_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: wheel3_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}

wheel4_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: wheel4_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}

wheel3_angle_controller:
  type: "position_controllers/JointPositionController"
  joint: wheel3_angle_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}

wheel4_angle_controller:
  type: "position_controllers/JointPositionController"
  joint: wheel4_angle_joint
  # pid: {p: 100.0, i: 0.01, d: 10.0}


/gazebo_ros_control:
  pid_gains:
      wheel1_joint:
          p: 100.0
          i: 1.0
          d: 0.1
      wheel2_joint:
          p: 100.0
          i: 1.0
          d: 0.1
      wheel3_joint:
          p: 100.0
          i: 1.0
          d: 0.1
      wheel4_joint:
          p: 100.0
          i: 1.0
          d: 0.1
      wheel3_angle_joint:
          p: 100.0
          i: 1.0
          d: 0.1
      wheel4_angle_joint:
          p: 100.0
          i: 1.0
          d: 0.1
